### PR TITLE
Draft changes for #585 - Clarify Base and Adjustment Rate Types

### PR DESCRIPTION
**Pull request by kirkycdr**

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -25775,31 +25780,16 @@ <h3 id="tocSproductdepositratetypedoc">Product Deposit Rate Types</h3></p>
...
-<tr>
-<td>BUNDLE_BONUS</td>
-<td>A bonus rate obtained by originating a bundle instead of a standalone product</td>
-<td>The name of the bundle</td>
-</tr>
-<tr>
 <td>FIXED</td>
 <td>Fixed rate for a period of time</td>
-<td>The period of time fixed. Formatted according to <a href="https://en.wikipedia.org/wiki/ISO_8601#Durations">ISO 8601 Durations</a></td>
+<td>The period of time fixed. Formatted according to ISO 8601 Durations</td>
```
Comment (nils-work) [inline docs/index.html:25785]: 'ISO 8601 Durations' should maintain link as per original text.

File: docs/index.html [hunk 2]
```diff
@@ -25824,53 +25814,64 @@ <h3 id="tocSproductlendingratetypedoc">Product Lending Rate Types</h3></p>
...
+<td>A bonus rate obtained by originating a bundle instead of a standalone product</td>
 <td>The name of the bundle</td>
 </tr>
 <tr>
-<td>CASH_ADVANCE</td>
-<td>Specific rate applied to cash advances from the account</td>
-<td>NA</td>
+<td>INTRODUCTORY</td>
+<td>An introductory bonus that will expire after a set period</td>
+<td>The period of time for the introductory rate. Formatted according to ISO 8601 Durations</td>
```
Comment (nils-work) [inline docs/index.html:25829]: 'ISO 8601 Durations' should maintain link as per original text.

File: docs/index.html [hunk 3]
```diff
@@ -25824,53 +25814,64 @@ <h3 id="tocSproductlendingratetypedoc">Product Lending Rate Types</h3></p>
...
 </tr>
+</tbody></table>
+
+<p><a id="productlendingratetypedoc"></a>
+<h3 id="tocSproductlendingratetypedoc">Product Lending Rate Types</h3></p>
+
+<p>Description of the usage of the lendingRateType field as it applies to products.</p>
+
+<p><a id="productlendingbaseratetypedoc"></a>
+<b id="tocSproductlendingbaseratetypedoc">Product Lending Base Rate Types</b></p>
```
Comment (nils-work) [inline docs/index.html:25839]: Remove 'Product '

File: docs/index.html [hunk 4]
```diff
@@ -25824,53 +25814,64 @@ <h3 id="tocSproductlendingratetypedoc">Product Lending Rate Types</h3></p>
...
+<tr>
+<td>CASH_ADVANCE</td>
+<td>Specific rate applied to cash advances from the account. This is expected to apply to products in the <code>CRED_AND_CHRG_CARDS</code> category only</td>
+<td>NA</td>
 </tr>
 <tr>
 <td>FIXED</td>
 <td>Fixed rate for a period of time</td>
-<td>The period of time fixed. Formatted according to <a href="https://en.wikipedia.org/wiki/ISO_8601#Durations">ISO 8601 Durations</a></td>
+<td>The period of time fixed. Formatted according to ISO 8601 Durations</td>
```
Comment (nils-work) [inline docs/index.html:25860]: 'ISO 8601 Durations' should maintain link as per original text.

File: docs/index.html [hunk 5]
```diff
@@ -25880,6 +25881,45 @@ <h3 id="tocSproductlendingratetypedoc">Product Lending Rate Types</h3></p>
 </tr>
 </tbody></table>
 
+<p><a id="productlendingadjustmentratetypedoc"></a>
+<b id="tocSproductlendingadjustmentratetypedoc">Product Lending Adjustment Rate Types</b></p>
```
Comment (nils-work) [inline docs/index.html:25885]: Remove 'Product '

File: docs/index.html [hunk 6]
```diff
@@ -25880,6 +25881,45 @@ <h3 id="tocSproductlendingratetypedoc">Product Lending Rate Types</h3></p>
...
+</tr>
+<tr>
+<td>DISCOUNT</td>
+<td>A specific discount rate that may be applied. A discount rate reduces the interest payable</td>
+<td>Description of the discount rate that is applicable</td>
+</tr>
+<tr>
+<td>INTRODUCTORY</td>
+<td>An introductory discount that will expire after a set period</td>
+<td>The period of time for the introductory rate. Formatted according to ISO 8601 Durations</td>
```
Comment (nils-work) [inline docs/index.html:25914]: 'ISO 8601 Durations' should maintain link as per original text.

File: slate/source/includes/banking/_product_components.md [hunk 7]
```diff
@@ -152,15 +152,29 @@ Description of the usage of the discountEligibilityType field as it applies to p
...
+<a id="productdepositadjustmentratetypedoc"></a>
+<ul><li id="tocSproductdepositadjustmentratetypedoc"><b>Deposit Adjustment Rate Types</b></li></ul>
+
+A product may have zero, one, or multiple adjustment rates that are taken to apply to a Base rate.
+
+Value | Description | Use of additionalValue Field
+-- | -- | --
+BONUS | A bonus rate available by meeting a specific criteria | A description of the criteria to obtain the bonus
+BUNDLE_BONUS | A bonus rate obtained by originating a bundle instead of a standalone product | The name of the bundle
+INTRODUCTORY | An introductory bonus that will expire after a set period | The period of time for the introductory rate. Formatted according to ISO 8601 Durations
```
Comment (nils-work) [inline slate/source/includes/banking/_product_components.md:177]: 'ISO 8601 Durations' should maintain link as per original text.

File: slate/source/includes/banking/_product_components.md [hunk 8]
```diff
@@ -152,15 +152,29 @@ Description of the usage of the discountEligibilityType field as it applies to p
...
-|MARKET_LINKED|A rate that is linked to a specific market, commodity or asset class|Details of the market linkage|
-|VARIABLE|A variable base rate for the product|NA|
+<a id="productdepositbaseratetypedoc"></a>
+<ul><li id="tocSproductdepositbaseratetypedoc"><b>Deposit Base Rate Types</b></li></ul>
+
+A deposit product is expected to present a single Base rate corresponding to relevant selection criteria including the rate `tiers` and `additionalValue`, where applicable.
+
+Value | Description | Use of additionalValue Field
+-- | -- | --
+FIXED | Fixed rate for a period of time | The period of time fixed. Formatted according to ISO 8601 Durations
```
Comment (nils-work) [inline slate/source/includes/banking/_product_components.md:162]: 'ISO 8601 Durations' should maintain link as per original text.

File: slate/source/includes/banking/_product_components.md [hunk 9]
```diff
@@ -169,19 +183,38 @@ Description of the usage of the depositRateType field as it applies to products.
...
-|FIXED|Fixed rate for a period of time|The period of time fixed. Formatted according to [ISO 8601 Durations](https://en.wikipedia.org/wiki/ISO_8601#Durations)|
-|FLOATING|A floating rate is relatively fixed but still adjusts under specific circumstances|Details of the float parameters|
-|INTRODUCTORY|An introductory discount that will expire after a set period|The period of time for the introductory rate.  Formatted according to [ISO 8601 Durations](https://en.wikipedia.org/wiki/ISO_8601#Durations)|
-|MARKET_LINKED|A rate that is linked to a specific market, commodity or asset class|Details of the market linkage|
-|PENALTY|A specific penalty rate that may be applied.  A penalty rate increases the interest payable|Description of the penalty rate that is applicable|
-|PURCHASE|Specific rate applied to purchases from the account|NA|
-|VARIABLE|A variable base rate for the product|NA|
+
+<a id="productlendingbaseratetypedoc"></a>
+<ul><li id="tocSproductlendingbaseratetypedoc"><b>Product Lending Base Rate Types</b></li></ul>
```
Comment (nils-work) [inline slate/source/includes/banking/_product_components.md:188]: Remove 'Product '

File: slate/source/includes/banking/_product_components.md [hunk 10]
```diff
@@ -169,19 +183,38 @@ Description of the usage of the depositRateType field as it applies to products.
...
+
+A lending product is expected to present a single Base rate corresponding to relevant selection criteria including the rate `tiers` and `additionalValue`, where applicable.
+
+Card products may have two or more base rates, including `CASH_ADVANCE` and `PURCHASE` as they may apply to different transaction types within an account. The `PURCHASE` lendingRateType is considered the rate commonly applicable to a card.
+
+Value | Description | Use of additionalValue Field
+-- | -- | --
+CASH_ADVANCE | Specific rate applied to cash advances from the account. This is expected to apply to products in the `CRED_AND_CHRG_CARDS` category only | NA
+FIXED | Fixed rate for a period of time | The period of time fixed. Formatted according to ISO 8601 Durations
```
Comment (nils-work) [inline slate/source/includes/banking/_product_components.md:197]: 'ISO 8601 Durations' should maintain link as per original text.

File: slate/source/includes/banking/_product_components.md [hunk 11]
```diff
@@ -169,19 +183,38 @@ Description of the usage of the depositRateType field as it applies to products.
...
+FLOATING | A floating rate is relatively fixed but still adjusts under specific circumstances | Details of the float parameters
+MARKET_LINKED | A rate that is linked to a specific market, commodity or asset class | Details of the market linkage
+PURCHASE | Specific rate applied to purchases from the account. This is expected to apply to products in the `CRED_AND_CHRG_CARDS` category only | NA
+VARIABLE | A variable base rate for the product | NA
+
+
+
+<a id="productlendingadjustmentratetypedoc"></a>
+<ul><li id="tocSproductlendingadjustmentratetypedoc"><b>Product Lending Adjustment Rate Types</b></li></ul>
```
Comment (nils-work) [inline slate/source/includes/banking/_product_components.md:206]: Remove 'Product '

File: slate/source/includes/banking/_product_components.md [hunk 12]
```diff
@@ -169,19 +183,38 @@ Description of the usage of the depositRateType field as it applies to products.
...
+
+A product may have zero, one, or multiple adjustment rates that are taken to apply to a Base rate.
+
+Value | Description | Use of additionalValue Field
+-- | -- | --
+BUNDLE_DISCOUNT_FIXED | A discount rate off the fixed rate obtained by originating a bundle instead of a standalone product | The name of the bundle
+BUNDLE_DISCOUNT_VARIABLE | A discount rate off the variable rate obtained by originating a bundle instead of a standalone product | The name of the bundle
+DISCOUNT | A specific discount rate that may be applied. A discount rate reduces the interest payable | Description of the discount rate that is applicable
+INTRODUCTORY | An introductory discount that will expire after a set period | The period of time for the introductory rate. Formatted according to ISO 8601 Durations
```
Comment (nils-work) [inline slate/source/includes/banking/_product_components.md:215]: 'ISO 8601 Durations' should maintain link as per original text.

File: docs/index.html (hunk 1 above)
Comment (JamesMBligh) [reply docs/index.html:25785]: I've fixed this (it will appear in index.html when we build on the main release branch)

**Review by JamesMBligh — APPROVED** [review]

**Pull request merged**